### DISCUSSION TITLE
Thanks for this

### DISCUSSION THREAD

**Discussion by isbkch** [SiddhantSadangi/st_login_form#15] [General]
I didn't use the full package since I'm not using Supabase for my app but I liked the form design and some of the ideas (allow guest) and ended up porting the code to my app. I just wanted to say thank you

**Comment by SiddhantSadangi** [comment]
Thanks for your kind words @isbkch 🤗 

What database are you using for your app? Extending support for other DBs (starting with those supported by [Streamlit's SQLConnection](https://github.com/SiddhantSadangi/st_login_form/pull/14)) is in the backlog

**Reply by isbkch** [reply]
Im using Streamlit for FE and FastApi for BE (including an Auth class). For DB im using a Postgres hosted on Neon

**Reply by SiddhantSadangi** [reply]
Ah.. since it is hosted on Neon, it'll need some more effort to integrate with the Login Form :(

However, is there any particular reason why you are not using Supabase? Supabase offers all functionalities you need (database, auth, etc), and I've built a [Supabase connector](https://github.com/SiddhantSadangi/st_supabase_connection) as well for Streamlit ;)

**Reply by isbkch** [reply]
Legacy code was built for Postgres with FastAPI and I’ve been changing the FE to use Streamlit widgets & components. Not sure I want to port the whole code to Supabase 😅 maybe if a biz need arises in the future 🤷🏻

**Reply by SiddhantSadangi** [reply]
Too much refactoring effort? 😅
